tests/pytests/test_issues.py

```diff
@@ -1088,3 +1088,14 @@ def test_unsafe_simpleString_values(env: Env):
 
   expected = [1, unsafe_value]
   env.expect('FT.SEARCH', unsafe_index, '*', 'SORTBY', unsafe_field, 'RETURN', 0).equal(expected)
+
```

Comment:
Nitpick (soon we'll have a formatter)
Why one line of separation here, compared to two in the other PRs?

Reply:
Just the way the conflict was resolved. I can add it back (but as you said, soon we will have a formatted)

Reply:
Na it's ok